Signed-off-by: dank@kegel.com

Raising the size of the hash table is a noticable win when linking 
at least one large app.  

There was a patch,
http://sources.redhat.com/ml/binutils/2004-06/msg00165.html
to do this dynamically.  No idea why that didn't make it in.
So this tiny change to raise the default size is just
a stopgap for now.


--- binutils-2.15/bfd/hash.c.old	2003-12-01 01:33:01.000000000 -0500
+++ binutils-2.15/bfd/hash.c	2006-03-01 16:26:26.701991000 -0500
@@ -295,7 +295,7 @@
 */
 
 /* The default number of entries to use when creating a hash table.  */
-#define DEFAULT_SIZE (4051)
+#define DEFAULT_SIZE (32749)
 
 /* Create a new hash table, given a number of entries.  */
 

